common_features.mk

```diff
@@ -199,7 +199,7 @@ endif
 
 
 ifeq ($(strip $(HD44780_ENABLE)), yes)
-    SRC += $(QUANTUM_DIR)/hd44780.c
+    SRC += drivers/avr/hd44780.c
```

Comment:
Not sure it matters, but the absolute path isn't necessary here.  It should be picked up just fine (like the ws2812.c and other drivers

Reply:
Yeah i wasn't sure so i added it. In my test project it compiled fine this way so i guess its staying.